packages/web/src/components/gcds-icon/gcds-icon.css

```diff
@@ -136,29 +136,81 @@
     margin-inline-end: var(--gcds-icon-margin-1000);
   }
 
+  /* Padding */
```

Comment:
Can you update this comment to say `Fixed width` instead of `padding` please since the prop changed?